pkg/rancher/migrations.go

```diff
@@ -540,3 +543,69 @@ func insertOrUpdateCondition(d data.Object, desiredCondition summary.Condition)
 
 	return true, nil
 }
+
+func migrateCAPIKubeconfigs(w *wrangler.Context) error {
+	logrus.Info("Running CAPI secret migration")
```

Comment:
nit: can this be more descriptive? i.e. `Running provisioningv2 CAPI kubeconfig secret migration`.

Reply:
Updated